src/rocksdb.rs

```diff
@@ -803,6 +803,23 @@ impl DB {
         Ok(())
     }
 
+    pub fn write_seq_opt(
+        &self,
+        batch: &WriteBatch,
+        writeopts: &WriteOptions,
+    ) -> Result<u64, String> {
+        let mut seq = 0;
+        unsafe {
+            ffi_try!(crocksdb_write_seq(
```

Comment:
what's the behavior of this

Reply:
Same with `write` but returns sequence number of writebatch